jib-core/src/integration-test/java/com/google/cloud/tools/jib/builder/BuildStepsIntegrationTest.java

```diff
@@ -42,25 +49,36 @@
 /** Integration tests for {@link BuildSteps}. */
 public class BuildStepsIntegrationTest {
 
+  /** Lists the files in the {@code resourcePath} resources directory. */
+  private static ImmutableList<Path> getFilesList(String resourcePath)
```

Comment:
If this is specifically for resources maybe it should be renamed to `getResourceFilesList`.